src/web/core/src/main/java/org/geoserver/web/data/layer/SQLViewAbstractPage.html

```diff
@@ -28,9 +28,12 @@
         </li>
     </ul>
 
-    <div class="button-group toolbar-sticky selfclear"><a href="#" class="form-button-save" wicket:id="save"><wicket:message
-        key="save">Save</wicket:message></a> <a href="#" class="form-button-cancel" wicket:id="cancel"><wicket:message
+    <div class="button-group toolbar-sticky selfclear">
+        <a href="#" class="form-button-save" wicket:id="save"><wicket:message key="save">Save</wicket:message></a>
+        <a href="#" class="form-button-save" wicket:id="back"><wicket:message key="back">Back</wicket:message></a>
```

Comment:
I am not sure this is consistent with our user interface form button naming. We tried to make the user interface consistent with:
* page:Apply, Save, Cancel
* nested-page: OK, Cancel

See http://geoserver.org/announcements/2020/09/26/geoserver-2-18-0-released.html